tests/unittests/config/test_cc_growpart.py

```diff
@@ -101,29 +99,36 @@ def __exit__(self, *args):
         pass
 
 
-class TestDisabled(unittest.TestCase):
-    def setUp(self):
-        super(TestDisabled, self).setUp()
-        self.name = "growpart"
-        self.cloud = None
-        self.args = []
+@pytest.fixture
+def disabled_context():
+    return {
+        "name": "growpart",
+        "cloud": None,
+        "args": [],
+        "handle": cc_growpart.handle,
+    }
 
-        self.handle = cc_growpart.handle
 
-    def test_mode_off(self):
-        # Test that nothing is done if mode is off.
+def test_mode_off(disabled_context):
+    # Test that nothing is done if mode is off.
 
-        # this really only verifies that resizer_factory isn't called
-        config = {"growpart": {"mode": "off"}}
+    # this really only verifies that resizer_factory isn't called
+    config = {"growpart": {"mode": "off"}}
+
+    with mock.patch.object(cc_growpart, "resizer_factory") as mockobj:
```

Comment:
We may be able to use pytest's mocker fixutre here instead of relying on mock.patch.object.